VRDR/IJEMortality.cs

```diff
@@ -116,6 +126,18 @@ public DeathRecord ToDeathRecord()
             return this.record;
         }
 
+       /// <summary>Validates this IJE string.  Throws an exception if it is invalid, otherwise returns true</summary>
+        public Boolean Valid()
```

Comment:
This is very nit-picky, but did you consider alternative names for this method? Perhaps `Validate()` or something like that to give the impression that this will _perform_ the validation when you actually call it (and if there are problems it will raise an exception, rather than return false which I think the name implies it might do).

Reply:
Might also make more sense as a `private` method instead.